src/share/import.js

```diff
@@ -0,0 +1,77 @@
+import fs from '../lib/fs';
+import Buffer from '../lib/buffer';
+import Path from '../lib/path';
+import WebTorrent from 'webtorrent';
+const sh = new fs.Shell();
+
+document.getElementById('torrentId').value = window.location.search.substring(

```

Comment:
Let's `npm install` https://github.com/webtorrent/magnet-uri and then we can take the string off the URL and make sure it's valid.  Also, same thing for the case that the user enters one manually on the page.  In both cases we should provide some kind of error if the URL isn't valid and we can't download.